README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires users to use Python 3. You can type `python --version` to check
+the current Python version on your device. If you do not have the right Python version,
+or you can not download or upgrade Python on your devices, you can download the
+[Pyenv](https://github.com/pyenv/pyenv) tool to set up a virtual environment for
+the newest Python version.
+
+To install Pyenv, you can use [Pyenv Installer](https://github.com/pyenv/pyenv-installer)
+by typing the command in terminal:
+
+```shell
+curl https://pyenv.run | bash
+```
+
+You will successfully install Pyenv. Please make sure that you have the
```

Comment:
The "you will successfully install" does not communicate the correct point. Is it more like "This step will then install Pyenv"?

Reply:
This has been fixed